regression/scripts/Vehicles/Boeing_737.py

```diff
@@ -30,7 +30,7 @@ def vehicle_setup():
     
     vehicle = SUAVE.Vehicle()
     vehicle.tag = 'Boeing_737800'    
-    
+    vehicle.configuration = 'Tube_Wing'
```

Comment:
This is not a minor change to SUAVE. This is kind of change requires a group discussion as this changes the ideology behind the code. We're going to need to need to discuss this next meeting if you want to make this type of change.

If you want to finish this pull request quickly I suggest you do this separately.

Reply:
So I realized this was a mistake. I removed it. The AVL script should read the fuselage configuration NOT vehicle configuration